src/api/app/helpers/webui/request_helper.rb

```diff
@@ -167,4 +167,18 @@ def list_maintainers(maintainers)
       user_with_realname_and_icon(maintainer, short: true)
     end.to_sentence.html_safe
   end
+
+  def descriptive_request_link(bs_request)
```

Comment:
You wrote in the commit message that this is going to be used in other areas of the code, but I don't see it here. Is this going to happen in another PR?

Reply:
This is not the method I was talking about.
I said the method [request_type_of_action](https://github.com/openSUSE/open-build-service/pull/12485/commits/9eba086259fb7da11c52765e6fcdcf46247c12af#diff-02694479bd751d8fea3a83a7d73e429e721178934e680726987b3856e15dd591R172) is going to be used from another part of the code,  so I move the helper from the component to a helper. This way, I can use it from the component and from the `descriptive_request_link` helper as well.

Reply:
Oh okay, I understand now :+1: